Directory.Packages.props

```diff
@@ -45,7 +45,7 @@
     <PackageVersion Include="System.Text.Encodings.Web" Version="4.7.2" />
 
     <!-- A conservative version of System.Text.Json must be used here since there is no backward compatibility guarantee during major version bumps. -->
-    <PackageVersion Include="System.Text.Json" Version="4.7.2" />
+    <PackageVersion Include="System.Text.Json" Version="[4.7.2, 8.0.0)" />
```

Comment:
Curious - any reason to make the change here?

Reply:
(sorry, I know this looks scary!) 
I stumbled across this issue https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605 when prototyping  and later found out this change is not required. Removed!